histomicstk/preprocessing/color_deconvolution/__init__.py

```diff
@@ -19,4 +22,7 @@
     'color_deconvolution',
     'complement_stain_matrix',
     'sparse_color_deconvolution',
+    'macenko_stain_matrix',
+    'snmf_stain_matrix',
+    'rgb_macenko_stain_matrix',
```

Comment:
I like the core method idea to appear in the name. So may be 

* 'deconvolve_stains_macenko_pca', 'deconvolve_stains_xu_snmf' or 
* 'unmix_stains_macenko_pca', 'unmix_stains_xu_snmf' or
* 'separate_stains_macenko_pca', 'separate_stains_xu_snmf'